source/PyMaterialX/PyBind11/include/pybind11/detail/common.h

```diff
@@ -103,7 +103,7 @@
 #  endif
 #  pragma warning(push)
 #  pragma warning(disable: 4510 4610 4512 4005)
-#  if defined(_DEBUG)
+#  if defined(_DEBUG) && !defined(Py_DEBUG)
```

Comment:
Fix was submitted back to PyBind11. See https://github.com/pybind/pybind11/pull/2025